cedars/app/api.py

```diff
@@ -0,0 +1,94 @@
+import os
+from loguru import logger
+from tenacity import retry, wait_exponential
+import requests
+from . import db
+
+def load_pines_url(project_id, superbio_api_token = None):
```

Comment:
can you define the expected return type in the function signature and also add documentation.